deps.edn

```diff
@@ -4,110 +4,110 @@
  ;; !!                                   PLEASE KEEP THESE ORGANIZED ALPHABETICALLY                                  !!
  ;; !!                                   AND ADD A COMMENT EXPLAINING THEIR PURPOSE                                  !!
  ;; !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
- {amalloy/ring-buffer                       {:mvn/version "1.3.1"               ; fixed length queue implementation, used in log buffering
+ {amalloy/ring-buffer                       {:mvn/version "1.3.1" ; fixed length queue implementation, used in log buffering
                                              :exclusions  [org.clojure/clojure
                                                            org.clojure/clojurescript]}
-  amalloy/ring-gzip-middleware              {:mvn/version "0.1.4"}              ; Ring middleware to GZIP responses if client can handle it
-  bigml/histogram                           {:mvn/version "4.1.4"}              ; Histogram data structure
```

Comment:
pretty big indentation diff here -- I'm worried this will cause merge conflicts in other PRs that bump deps or whatever. Maybe worth reverting it

We do have a `comment-column` declaration here

https://github.com/metabase/metabase/blob/4f09fa3b631f882a3c5edcab4508769ffb20d4fa/deps.edn#L1

so not sure why it did this to you 😢

Reply:
woah. inadvertent. good catch